src/pages/[platform]/build-a-backend/data/connect-to-existing-data-sources/connect-postgres-mysql-database/index.mdx

```diff
@@ -195,7 +195,7 @@ const { data: events } = await client.models.event.list()
 
 When deploying your app to production, you need to [add the database connection string as a secret](/[platform]/deploy-and-host/fullstack-branching/secrets-and-vars/#set-secrets). Make sure to add the appropriate database connection string with the same secret name you used in the sandbox environment. For example, we used `SQL_CONNECTION_STRING` above.
 
-<video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true}>
+<video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true} ariaDescription="Video - Configuring database connection for production">
```

Comment:
Can we switch this to use the docs <Video> component instead of the <video> element